pom.xml

```diff
@@ -139,6 +150,29 @@
                 </configuration>
             </plugin>
         </plugins>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.jvnet.jaxb2.maven2</groupId>
+                    <artifactId>maven-jaxb2-plugin</artifactId>
+                    <version>${maven-jaxb2-plugin.version}</version>
+                    <configuration>
+                        <plugins combine.children="append">
+                            <plugin>
+                                <groupId>org.jvnet.jaxb2_commons</groupId>
+                                <artifactId>jaxb2-basics</artifactId>
+                                <version>${jaxb2.version}</version>
```

Comment:
✏️ Maybe we could make this a little more distinct from `jaxb.version` by naming it `jvnet.jaxb2.version`?

Reply:
Good idea, I'm also gonna check to see if there's consistency issues in naming between here and DDF with anything I added in.